experiments/modelfree/plot_noisy_observations.py

```diff
@@ -0,0 +1,141 @@
+import json
```

Comment:
Should this live in `experiments/modelfree/` or in e.g. `src/modelfree/visualize/`? I've largely been using `experiments/` as a directory for ad-hoc, stand-alone scripts. I think could make a case for either here, but I think leaning towards including it in the main `src` directory, especially if we integrate it more with the rest of `modelfree`.

Reply:
Yeah, I didn't really know what your internal heuristic for that here, but it did seem more like post-experiment analysis than actual source code used in an experiment, which was my logic for putting it here.